Sources/Public/Item Views/Shape.swift

```diff
@@ -14,9 +14,39 @@
 // limitations under the License.
 
 import CoreGraphics
+import UIKit.UIBezierPath
 
 /// Represents a background or highlight layer shape; used by `DayView` and `DayOfWeekView`.
-public enum Shape: Hashable {
+public enum Shape {
   case circle
   case rectangle(cornerRadius: CGFloat = 0)
+  case custom((CGRect) -> UIBezierPath)
+}
+
+extension Shape: Hashable {
+  public func hash(into hasher: inout Hasher) {
+    switch self {
+    case .circle:
+      break
+    case .rectangle(let cornerRadius):
+      hasher.combine(cornerRadius)
+    case .custom:
+      break
```

Comment:
not sure if I should be adding any kind of hash based on closure

Reply:
Yeah, this will cause issues with view reuse since closures can't be compared for equality.